src/smc-hub/postgres-server-queries.coffee

```diff
@@ -105,6 +105,52 @@ class exports.PostgreSQL extends PostgreSQL
                 'time       :: TIMESTAMP' : 'NOW()'
             cb     : opts.cb
 
+    webapp_error: (opts) =>
+        opts = defaults opts,
+            account_id   : undefined
+            name         : undefined
+            message      : undefined
+            comment      : undefined
+            stacktrace   : undefined
+            path         : undefined
+            lineNumber   : undefined
+            columnNumber : undefined
+            severity     : undefined
+            browser      : undefined
+            mobile       : undefined
+            responsive   : undefined
+            user_agent   : undefined
+            smc_version  : undefined
+            build_date   : undefined
+            smc_git_rev  : undefined
+            uptime       : undefined
+            start_time   : undefined
+            cb           : undefined
+        @_query
+            query       : 'INSERT INTO webapp_errors'
+            values      :
+                'id            :: UUID'      : misc.uuid()
+                'account_id    :: UUID'      : opts.account_id
+                'name          :: TEXT'      : opts.name
+                'message       :: TEXT'      : opts.message
+                'comment       :: TEXT'      : opts.comment
+                'stacktrace    :: TEXT'      : opts.stacktrace
+                'path          :: TEXT'      : opts.path
```

Comment:
good question, let me read up on this a bit. right now it's like `path: https://cloud.sagemath.com/projects/.../files/test/`